Backends/Graphics4/Direct3D9/Sources/Kore/VertexBufferImpl.cpp

```diff
@@ -12,7 +12,7 @@ Graphics4::VertexBuffer* VertexBufferImpl::_current = nullptr;
 
 VertexBufferImpl::VertexBufferImpl(int count, int instanceDataStepRate) : myCount(count), instanceDataStepRate(instanceDataStepRate) {}
 
-Graphics4::VertexBuffer::VertexBuffer(int count, const VertexStructure& structure, int instanceDataStepRate) : VertexBufferImpl(count, instanceDataStepRate) {
+Graphics4::VertexBuffer::VertexBuffer(int count, const VertexStructure& structure, usage: Usage, int instanceDataStepRate) : VertexBufferImpl(count, instanceDataStepRate) {
```

Comment:
`usage: Usage` should probably be `Usage usage` as we're not in haxe land anymore?